net-dns/letsdns/letsdns-1.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="Manage DANE TLSA records in DNS servers"
+HOMEPAGE="https://letsdns.org"
+SRC_URI="https://github.com/LetsDNS/letsdns/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="dev-python/cryptography[${PYTHON_USEDEP}]
+	dev-python/dnspython[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]"
+
+src_prepare() {
+	eapply_user
+	echo -e '[options.packages.find]\nexclude=tests*' >>setup.cfg || die
```

Comment:
Please do not forget to also fix this upstream at https://github.com/LetsDNS/letsdns/blob/master/setup.cfg

Reply:
"There's a brach for this." 😉